hack/api-docs/build.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License;
+# you may not use this file except in compliance with the Elastic License.
+
+# Script to generate API reference documentation from the source code.
+
+set -euo pipefail
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+DOCS_DIR="${SCRIPT_DIR}/../../docs"
+REFDOCS_PKG="github.com/elastic/gen-crd-api-reference-docs"
+REFDOCS_VER="master"
+REFDOCS_BIN="$(go env GOPATH)/bin/$(basename $REFDOCS_PKG)"
+
+install_refdocs() {
+    local INSTALL_DIR=$(mktemp -d)
+    (
+        cd $INSTALL_DIR
+        go mod init github.com/elastic/eck-refdocs
+        go get -u "${REFDOCS_PKG}@${REFDOCS_VER}"
```

Comment:
I got an error:
```shell
> m generate-api-docs
hack/api-docs/build.sh
...
+ install_refdocs
++ mktemp -d
+ local INSTALL_DIR=/tmp/tmp.gPCntWTtm0
+ cd /tmp/tmp.gPCntWTtm0
+ go mod init github.com/elastic/eck-refdocs
+ go get -u github.com/elastic/gen-crd-api-reference-docs@master
go: cannot use path@version syntax in GOPATH mode
Makefile:89: recipe for target 'generate-api-docs' failed
make: *** [generate-api-docs] Error 1
```
Solved with `export GO111MODULE=on`. Do you also have `GO111MODULE` set to `on`? So is it a requirement?